### DISCUSSION TITLE
Picklist items deleted: On using Ctrl+A to select all, then right clicking on item it gets removed from the list

### DISCUSSION THREAD

**Discussion by amura** [primefaces/community#1412] [PrimeNG] [answered]
We have found an issue with the picklist. If you have some items in either the source or target tables, and then press Ctrl+A to select all items in the list, if you then go and right click on a selected item in the list, this item will get removed from the list altogether. The only way to restore the picklist items is by refreshing the component. This issue can be seen in the sample here https://primeng.org/picklist#filter

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting the issue, we'll address this in upcoming releases. Issue link: https://github.com/primefaces/primeng/issues/15046
